pom.xml

```diff
@@ -58,30 +58,30 @@
 		<maven-eclipse-plugin.version>2.10</maven-eclipse-plugin.version>
 		<!-- for backwards compatibility, use maven-checkstyle-plugin.version -->
 		<checkstyle.version>${maven-checkstyle-plugin.version}</checkstyle.version>
-		<puppycrawl-tools-checkstyle.version>8.29</puppycrawl-tools-checkstyle.version>
+		<puppycrawl-tools-checkstyle.version>8.43</puppycrawl-tools-checkstyle.version>
 		<spring-javaformat.version>0.0.26</spring-javaformat.version>
-		<maven-failsafe-plugin.version>2.22.2</maven-failsafe-plugin.version>
-		<maven-javadoc-plugin.version>3.2.0</maven-javadoc-plugin.version>
+		<maven-failsafe-plugin.version>3.0.0-M5</maven-failsafe-plugin.version><!-- @releaser:version-check-off -->
+		<maven-javadoc-plugin.version>3.3.0</maven-javadoc-plugin.version>
 		<maven-source-plugin.version>3.2.1</maven-source-plugin.version>
 		<maven-jar-plugin.version>3.2.0</maven-jar-plugin.version>
 		<maven-antrun-plugin.version>3.0.0</maven-antrun-plugin.version>
-		<maven-surefire-plugin.version>2.22.2</maven-surefire-plugin.version>
+		<maven-surefire-plugin.version>3.0.0-M5</maven-surefire-plugin.version><!-- @releaser:version-check-off -->
 		<maven-surefire-report-plugin.version>3.0.0-M5</maven-surefire-report-plugin.version><!-- @releaser:version-check-off -->
 		<maven-war-plugin.version>3.3.1</maven-war-plugin.version>
 		<exec-maven-plugin.version>3.0.0</exec-maven-plugin.version>
 		<maven-resources-plugin.version>3.2.0</maven-resources-plugin.version>
-		<git-commit-id-plugin.version>4.0.4</git-commit-id-plugin.version>
+		<git-commit-id-plugin.version>4.0.5</git-commit-id-plugin.version>
```

Comment:
fyi, there is a 5.x version of the plugin with new group and artifact id: https://github.com/git-commit-id/git-commit-id-maven-plugin#relocation-of-the-project

Reply:
I wish they could fit git more in the new coordinates :-|